db/sql/sql_test.go

```diff
@@ -52,7 +52,7 @@ var _ = Describe("Sql", func() {
 	BeforeEach(func() {
 		var dbType string
 		if os.Getenv("MYSQL_TEST") == "true" {
-			conn = "gohan:gohan@/gohan_test"
+			conn = "root@/gohan_test"
```

Comment:
Why sometimes we are forcing a TCP connection and sometimes (here) we're not?

Reply:
changed connections to use tcp